molgenis-js/src/main/resources/js/math.min.js

```diff
@@ -1,3 +1,4 @@
+"use strict";
```

Comment:
math.js [4.3.0](http://mathjs.org/history.html) has breaking changes compared to 2.1.1. Is this desirable?

Reply:
I strongly suspect it is only in use indirectly by MagmaScript unit conversions. Updating release notes.